packages/node_modules/@webex/plugin-meetings/src/config.js

```diff
@@ -18,13 +18,8 @@ export default {
       detection: true,
       // Timeout duration to wait for ICE to reconnect if a disconnect is received.
       iceReconnectionTimeout: 10000,
-      retry: {
-        times: 2,
-        backOff: {
-          start: 1000,
-          rate: 2
-        }
-      }
+      // Amount of times attempting to rejoin a meeting during reconnect
+      maxRejoinAttempts: 3
```

Comment:
Was this changed since back-off logic already exists elsewhere?

Reply:
yes, this amount is specific to just the rejoin logic portion of the reconnect function.